BreanaMarie/tic-tac-toe/ticTacToe.js

```diff
@@ -0,0 +1,458 @@
+    $("p").html(who);
+});
+
+//$("#turnEdit").html("It is " + round +"'s Turn");
+
+//check if box is free or has been played
+function changeSpace(selectedSpace){
+	turn();
+	$('div.box').children().on('click', function(){
+		console.log($(this).text()+ " has been clicked");
+		selectedSpace = $(this);
+		console.log(selectedSpace);
+
+		//if freeSpace
+		if (selectedSpace.hasClass('freeSpace')){
+			console.log("the space is free");
+
+			//check turn
+			if (round ==='WDI 23'){
+				console.log("WDI 23 just went");
+				//change space to played
+			    if(selectedSpace.hasClass('freeSpace')) { 
+			        selectedSpace.removeClass('freeSpace');
+			        selectedSpace.addClass('playedSpaceX');
+			        //check if play won
+			        winSpaces();
+			        console.log (i);
+			        i ++;
+			        turn();
+		    	}	
+			}
+		    else if (round === 'Space Jammer'){
+		    	console.log("Space Jammer just went");
+		    	if(selectedSpace.hasClass('freeSpace')) { 
+			        selectedSpace.removeClass('freeSpace');
+			        selectedSpace.addClass('playedSpaceO');
+			        
+			        winSpaces();
+			        console.log(i);
+			        i ++;
+			        turn();
+		    	} 
+		    }
+		}
+		//if already playedSpace*
+		else if (selectedSpace.hasClass('playedSpaceX' || 'playedSpaceO')){
+			alert("this spot has already been played");
+			changeSpace();
+		}	
+	});
+}
+changeSpace();
+
+//create a reset button
+var resetSpaces =function(){
+	console.log("board has been reset");
+	$('div[name="space"]').removeClass('playedSpaceX');
+	$('div[name="space"]').removeClass('playedSpaceO');
+	$('div[name="space"]').removeClass('winnerSpace');
+	$('div[name="space"]').addClass('freeSpace');
+	round='WDI 23';
+	i=0;
+	$("p").html("The Game Begins with WDI 23");
+};
+
+//listen for reset button to be clicked
+$(".buttonReset").on('click', function(){
+	resetSpaces();
+});
+
+
+
+//returns which way was used to win
+var winSpaces = function(){
+	return (acrossTop() || acrossMid() || acrossBot()|| diagLeft() || diagRight() || downFirst() || downMid() || downLast());
+};
+
+//check three in a row change background color to show win
+//list ways to win
+//declare winner and reset
+var acrossTop= function($box1, $box2, $box3){
+	var space1 = $('#box1').attr("class");
+	var space2 = $('#box2').attr("class");
+	var space3 = $('#box3').attr("class");
+
+	if ((space1 === space2) && (space2 === space3)){
+		if (space1 === 'playedSpaceX'){
+			console.log ('X has won');
+			$('#box1').addClass('winnerSpace');
+			$('#box2').addClass('winnerSpace');
+			$('#box3').addClass('winnerSpace');
+			alert(round + " Has Won");
+			resetSpaces();
+			return acrossTop;
+		}
+		else if (space1 === 'playedSpaceO'){
+			console.log ('O has won');
+			$('#box1').addClass('winnerSpace');
+			$('#box2').addClass('winnerSpace');
+			$('#box3').addClass('winnerSpace');
+			alert(round + " Has Won");
+			resetSpaces();
+			return acrossTop;
```

Comment:
This is another place where the logic is very similar.  You could just detect when they're the same and then later after determining that detect which player is the winner.